core/controllers/suggestion.py

```diff
@@ -148,6 +148,15 @@ def post(self):
 
         self.render_json(self.values)
 
+    @acl_decorators.can_suggest_changes
```

Comment:
This acl will allow any contributors to edit other's suggestions, is that expected?

Reply:
Introduced a new decorator.